backend/corpora/dataset_processing/process.py

```diff
@@ -284,15 +283,19 @@ def _get_term_pairs(base_term):
 
     metadata = {
         "name": adata.uns["title"],
-        "organism": {"label": adata.uns["organism"], "ontology_term_id": adata.uns["organism_ontology_term_id"]},
+        "organism": _get_term_pairs("organism"),
         "tissue": _get_term_pairs("tissue"),
         "assay": _get_term_pairs("assay"),
         "disease": _get_term_pairs("disease"),
-        "sex": list(adata.obs.sex.unique()),
+        "sex": _get_term_pairs("sex"),
         "ethnicity": _get_term_pairs("ethnicity"),
         "development_stage": _get_term_pairs("development_stage"),
         "cell_count": adata.shape[0],
         "mean_genes_per_cell": numerator / denominator,
+        "is_primary_data": adata.obs["is_primary_data"].unique()[0],
```

Comment:
TODO: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/1356. Will be done in a separate PR